client/blockchain-service/src/handler.rs

```diff
@@ -348,6 +348,44 @@ impl Actor for BlockchainService {
                         }
                     }
                 }
+                BlockchainServiceCommand::GetProviderId {
```

Comment:
Are you using this for something?

Reply:
Yes

https://github.com/Moonsong-Labs/storage-hub/blob/bc1ad6c6e229a71df12ea6e48c9c396a5d4e2eee/node/src/tasks/bsp_upload_file.rs#L363-L369